.github/workflows/release_droid_prepare_original_checksum.yml

```diff
@@ -0,0 +1,69 @@
+name: Release Droid - Prepare Original Checksum
+
+on:
+  workflow_dispatch:
+
+jobs:
+  prep-testbed:
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - id: set-matrix
+        run: |
+          sudo apt-get install jq
+          sudo bash ./tests/scripts/cran/install_cran_repos.sh
+          echo "Result:"
+          echo $(sudo bash ./tests/scripts/cran/get_R_releases.sh)
+          echo "::set-output name=matrix::$(sudo bash ./tests/scripts/cran/get_R_releases.sh)"
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+
+  integration_tests:
+    needs: prep-testbed
+    environment: publish
+    strategy:
+      fail-fast: false
+      matrix:
+        r-version: ${{fromJson(needs.prep-testbed.outputs.matrix)}}
+    runs-on: ubuntu-18.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Checkout test environment
+        run: git clone https://github.com/exasol/integration-test-docker-environment.git
+        working-directory: ..
+      - name: Spawn EXASOL environemnt
+        run: ./start-test-env spawn-test-environment --environment-name test --database-port-forward 8888 --bucketfs-port-forward 6666 --db-mem-size 4GB
+        working-directory: ../integration-test-docker-environment
+      - name: Docker login
+        run: echo "$SECRET_DOCKER_TOKEN" | docker login --username "$SECRET_DOCKER_USER_NAME" --password-stdin
+        env: # Set the secret as an input
+          SECRET_DOCKER_USER_NAME: ${{ secrets.DOCKER_USER_NAME }}
+          SECRET_DOCKER_TOKEN: ${{ secrets.DOCKER_TOKEN }}
+      - name: Run all tests
+        run: |
+          ./tests/scripts/pull_docker_image.sh ${{ matrix.r-version }}
+          ./tests/scripts/execute_docker_test_env.sh ${{ matrix.r-version }}
+
+  publish-main:
```

Comment:
The main CI gihub workflow also has a publish step, do we need it there still?

Reply:
Yes, this step pushes a new docker image to docker hub (which then is used by the CI build itself).